cc-backend/readme.md

```diff
@@ -0,0 +1,7 @@
+- Clone from repo.
```

Comment:
some mention of the technologies used, like "Express Typescript server"
formatting guide: https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax

Reply:
Updated readme with some technologies and versions. Should discuss together what and add there and how we format them so every readme would look similar.